### PR TITLE
Update the grant application details and info around Open Collective

### LINKED ISSUE #110: Ladies Grant Application Guidelines for R Consortium Grants
Create more guidelines around R Consortium Grants and how they can use Open collective in the guide.

### PR DESCRIPTION
**Pull request by BatoolMM**
This PR is to enhance the availability of documentation and providing guidance to R-Ladies chapter organizers regarding the application process for the RUGS grant and the establishment of an Open Collective account for efficient fund management

Fix #110

### CODE REVIEW

File: content/organization/intro/finance/index.en.md [hunk 1]
```diff
@@ -61,6 +61,35 @@ Now as regards to the subject of the email, you want it to
...
+     - The desired grant amount.
+     - The R-Ladies chapter for which you are applying.
+     - The name of the organizer(s) in your chapter who will be responsible for managing the grant.
+
+**Step 2: Complete the Online Application**
+3. Once Batool Almarzouq has received and recorded the above information, proceed to the grant application process. Access the grant application form [available on the R Consortium website](https://www.r-consortium.org/projects/r-user-group-support-program).
+   - Fill out all required fields accurately and comprehensively.
+   - Ensure that you have prepared a detailed proposal outlining the purpose and scope of your project.
+   - Submit the completed application form through the website.
+
```
Comment (cosimameyer) [inline content/organization/intro/finance/index.en.md:78]: It looks like there is a "Step 3" missing, isn't it?

**Review by cosimameyer — COMMENTED** [review]
Besides the missing step 3 (or renumbering the existing steps), it looks good to me! ☺️ I'm not sure if "bypassing" the three steps is on purpose (if it is, this also looks good to me. I just haven't reviewed a PR for the `rladiesguide` yet ☺️)

File: content/organization/intro/finance/index.en.md (hunk 1 above)
Comment (BatoolMM) [reply content/organization/intro/finance/index.en.md:79]: I fixed this - thank you so much!

**Comment by yabellini** [comment]
Hi, I was wondering if we could avoid using Batool's email to generate an R-Ladies email and forward it to Batool.  We have been removing all personal mentions and emails on the different roles to make it easier when team members change. 

@rladies/leadership, what do you think?  Sorry if we discussed this before and I missed it.

**Comment by shannonpileggi** [comment]
> Hi, I was wondering if we could avoid using Batool's email to generate an R-Ladies email and forward it to Batool. We have been removing all personal mentions and emails on the different roles to make it easier when team members change.
> 
> @rladies/leadership, what do you think? Sorry if we discussed this before and I missed it.

this was not yet discused and this is a very good idea @yabellini

**Comment by shannonpileggi** [comment]
@BatoolMM sorry i don't know. is your role just administrative support, or do you provide advice on the actual submission as well?

**Comment by BatoolMM** [comment]
Yes, I agree on not using personal email as it makes it easier when there is a change in the team.

@shannonpileggi my role is administrative support where I track the submission and keep record of it.

**Comment by shannonpileggi** [comment]
thanks for clarifying, @BatoolMM! if we had a mechanism for advice i would have suggested including that; otherwise, it is good as it is.

for the remaining changes, we need either @yabellini or @averi-giudicessi to create a general email address to use for this that forwards to you, and then once we have that in place as far as i am concerned it is good to merge.

**Comment by drmowinckels** [comment]
what is the status here? @averi-giudicessi and @yabellini, I think we just need an email for this task, and then we are good to get it merged?

**Comment by averi-giudicessi** [comment]
Hi @BatoolMM so sorry for the delay on this! I went ahead and created the email grants@rladies.org, I realized that you don't have an rladies email account to set up the forwarding. Can you let me know what you like your username to be for the email account and I can go ahead and get that set up. Thanks!

**Comment by BatoolMM** [comment]
thank you @averi-giudicessi - much appreciated! 
I would like to be forwarded balmarzouq@turing.ac.uk if that's possible - if you still need username, you can use my name batool. Let me know if you need anything else!

**Comment by averi-giudicessi** [comment]
I forwarded the grants email to Batool's email address so we should be all set!

**Comment by yabellini** [comment]
thanks a lot @averi-giudicessi for taking care of the email side of this 🙏

**Comment by BatoolMM** [comment]
Thank you @cosimameyer @yabellini @shannonpileggi @averi-giudicessi for reviewing this PR- I just changed the email in the text to [grants@rladies.org](mailto:grants@rladies.org), happy to merge if you are all comfortable!

**Comment by yabellini** [comment]
Here are more info about this:

> Regarding the announcement to dissolve Open Collective Foundation
> by Lauren Gardner
> Dear OSC community,
> You may be aware of recent developments regarding the dissolution of Open Collective Foundation (OCF) by the end of the year 2024. We are extremely sad to hear the news and are still processing it, but we wanted to send a quick note to our collectives. Please be assured Open Source Collective is not affected and will continue to operate as usual. Our organizations are distinct entities, each with its own budget, accounts, staff, board of directors, and mission.
> The similarity in our names stems from a common origin, as the founders of Open Source Collective were also involved in creating the fiscal hosts Open Collective Foundation and Open Collective Europe. Additionally, we all utilize the Open Collective platform, a detail we recognize may contribute to confusion. Although the naming of these entities is similar, we are all separate business entities.
> Open Source Collective remains committed to our mission and to serving the open source community.
> We have compiled a list of OSC-hosted collectives with an entity or grant currently hosted by OCF. We will discuss these with OCF and reach out individually regarding the next steps. If you are an OSC-hosted collective with a project on OCF and want to contact us directly with questions or have yet to hear from us by the end of this week, please get in touch with us at [hello@oscollective.org](mailto:hello@oscollective.org).
> For collectives currently hosted by Open Collective Foundation, you may be curious to know more about Open Source Collective and our fiscal hosting offering. Open Source Collective is a 501(c)(6) nonprofit fiscal host that only hosts open source software-related projects. If your collective works in the open source community, you may be eligible to be hosted by us. However, it is essential to be aware that as a 501(c)(6) organization, there may be restrictions on funding opportunities from specific donors who are limited to funding 501(c)(3) entities. Additionally, there may be constraints on transferring funds from a 501(c)(3) fiscal host to a 501(c)(6) fiscal host.
> If you are an organization looking for a new 501(c)(3) fiscal host and are a social movement or charitable organization, you may wish to reference a list of fiscal host organizations here: https://www.fiscalsponsors.org/member-directory.
> Our hearts are heavy for our friends and colleagues at Open Collective Foundation with whom we shared dreams, efforts, admiration, and inspiration as we each worked to build a community of care and support.
> Please let us know how we can help and if there are any questions we can help you answer.
> Lauren Gardner, Executive Director of Open Source Collective

**Comment by shannonpileggi** [comment]
@BatoolMM is it ok if we remove the open collective stuff on your behalf or would you like to do so?

File: content/organization/intro/finance/index.en.md [hunk 2]
```diff
@@ -61,6 +61,36 @@ Now as regards to the subject of the email, you want it to
...
+3. Once Batool Almarzouq has received and recorded the above information, proceed to the grant application process. Access the grant application form [available on the R Consortium website](https://www.r-consortium.org/projects/r-user-group-support-program).
+   - Fill out all required fields accurately and comprehensively.
+   - Ensure that you have prepared a detailed proposal outlining the purpose and scope of your project.
+   - Submit the completed application form through the website.
+   - contact Batool again to confirm submission and report the day you submitted your application at.
+
+**Step 3: Fund Disbursement**
+4. Upon approval of your grant application by R Consortium, you will receive the grant funds. You have the flexibility to transfer the funds to any suitable account for your chapter's use.
+
+**Optional: Use Open Collective for Transparency**
```
Comment (yabellini) [inline content/organization/intro/finance/index.en.md:83]:
```suggestion

```

File: content/organization/intro/finance/index.en.md [hunk 3]
```diff
@@ -61,6 +61,36 @@ Now as regards to the subject of the email, you want it to
...
+5. For a more transparent financial management approach, consider using [Open Collective](https://opencollective.com/) to manage your grant funds.
```
Comment (yabellini) [inline content/organization/intro/finance/index.en.md:84]:
```suggestion

```

File: content/organization/intro/finance/index.en.md [hunk 4]
```diff
@@ -61,6 +61,36 @@ Now as regards to the subject of the email, you want it to
...
+   - Contact [Open Collective](https://opencollective.com/) to open an account for your R-Ladies chapter.
```
Comment (yabellini) [inline content/organization/intro/finance/index.en.md:85]:
```suggestion

```

File: content/organization/intro/finance/index.en.md [hunk 5]
```diff
@@ -61,6 +61,36 @@ Now as regards to the subject of the email, you want it to
...
+   - After creating your Open Collective account, invite the R-Ladies Global Team account to be co-administrator for the grant.
```
Comment (yabellini) [inline content/organization/intro/finance/index.en.md:86]:
```suggestion

```

**Review by yabellini — APPROVED** [review]
Removing open collective mention.

**Comment by yabellini** [comment]
Thanks a lot, @BatoolMM; I  moved forward with the content update to have your improvements to the guide available for our chapter organizers.

**Pull request merged**